packages/codemods/react/19/replace-act-import/src/index.ts

```diff
@@ -101,6 +101,9 @@ export default function transform(
 
       const existingReactImportCollection = root.find(j.ImportDeclaration, {
         source: { value: "react" },
+        importKind(value) {
+          return value === "value" || value == null;
```

Comment:
eqeqeq (`===`), other than that LGTM, thanks
can you bump the patch version in `.codemodrc.json`, please?

Reply:
The type of `importKind` is `"type" | "value" | "typeof" | undefined` so the `==` was to also match undefined. 

I'm honestly not sure if realistically any parser used with jscodeshift would create an AST with an empty `importKind`